service/src/main/java/uk/gov/hmcts/reform/fpl/service/CaseManagementOrderProgressionService.java

```diff
@@ -56,6 +58,7 @@ private void progressDraftCaseManagementOrder(CaseDetails caseDetails, CaseManag
                 break;
             case PARTIES_REVIEW:
                 caseDetails.getData().put(CASE_MANAGEMENT_ORDER_SHARED.getKey(), order.getOrderDoc());
+                publishReadyForPartyReviewEvent(caseDetails, requestData);
```

Comment:
Might be worth thinking about verifying `applicationEventPublisher` calls in unit tests. Happy to leave is as it is though, because I can see that we're not doing it for any route currently. But it feels like this class is doing too much to test it properly and some parts could potentially be extracted.

Reply:
Good to think about, we can add it to tech improvements perhaps (looks like we could verify applicationEventPublisher in several tests where we currently don't)